src/DetailsView/components/base-visual-helper-toggle.tsx

```diff
@@ -18,13 +17,13 @@ export abstract class BaseVisualHelperToggle extends React.Component<VisualHelpe
 
         return (
             <div className="visual-helper">
-                <div className="visual-helper-text">Visual helper</div>
+                <div className="visual-helper-text">Highlight instances on target page</div>
```

Comment:
I'd prefer if we are using this in multiple places to just keep it as a constant somewhere that we can access.

Reply:
My first intention was to just make this change without making more substantial code changes. But I guess it makes sense for me to make this change.

Reply:
It's just easier for the test changes in the future, I think. Thanks! 😄

Reply:
Let me know if the current change feels better to you.